### DISCUSSION TITLE
chakra-cli command isn't recognized after installing @chakra-ui/cli

### DISCUSSION THREAD

**Discussion by eli-s-r** [chakra-ui/chakra-ui#4712] [Q&A] [answered]
After running `npm install -D @chakra-ui/cli` as specified in the [docs](https://chakra-ui.com/docs/theming/advanced#install), trying to use `chakra-cli` yields:
```chakra-cli: command not found```

I also tried installing it globally, and as a normal dependency instead of a dev dependency.

Running the command through npx as `npx chakra-cli ...` doesn't crash, but the types aren't appearing in autocomplete in VSCode.

Any ideas? Thanks!

**Comment by stevaras** [comment]
Although it's been a couple years, not sure why your chakra-cli command is not recognized, but as far as the VSCode autocomplete, they now suggest to restart your typescript server. 
Check inside your node_modules\@chakra-ui\styled-system\dist\theming.types.d.ts to check if the updates are actually there, and then press Ctrl + Shift + P and on the opened prompt choose "TypeScript: Restart TS Server"

**Comment by riyajath-ahamed** [comment]
@eli-s-r Try Using 
`npx @chakra-ui/cli snippet add <component>`

**Comment by segunadebayo** [comment, marked as answer]
The correct command is either

- Run from npx: `npx @chakra-ui/cli snippet add <component>`
- Run from local: `npx chakra snippet add <component>`

For now, we recommended installing the CLI as a dev deps and running it from local

**Reply by riyajath-ahamed** [reply]
Hi @segunadebayo ,
But the `npx chakra snippet add <component>` command produces an error: chakra-cli: command not found. It seems this command should be **globally** available rather than limited to a **devDependency**.

**Comment by Adams-Ijachi** [comment]
Hello @segunadebayo  when i try `npx @chakra-ui/cli snippet add file-upload`  i get this error `●  Adding 1 snippet(s)...
│
◇  Writing selected snippets
SyntaxError: Unexpected token '<', "<!DOCTYPE "... is not valid JSON`

i am new to chakra so not sure if my setup is wrong or something

**Reply by cucunber** [reply]
try `npx @chakra-ui/cli snippet add file-button`
all compositions are allowed by file name (see here https://github.com/chakra-ui/chakra-ui/tree/main/apps/compositions/src/ui)
chakra docs says `file-upload`, but where is no such file, so it breaks

**Reply by bibhashkar** [reply]
Thanks, that solved my problem
